src/libraries/System.Runtime/tests/System/ActivatorTests.cs

```diff
@@ -621,11 +621,9 @@ public void CreateInstance_PublicOnlyValueTypeWithPrivateDefaultConstructor_Thro
         [Theory]
         [ActiveIssue("https://github.com/dotnet/runtime/issues/34030", TestPlatforms.Linux | TestPlatforms.Browser, TargetFrameworkMonikers.Netcoreapp, TestRuntimes.Mono)]
```

Comment:
Can this ActiveIssue be remove as well?

Reply:
Possibly? I'm not sure if the original issue's platform and Browser are case sensitive as well. @MaximLipnin  @safern ?
I'll try to build for Browser now, but its not clear what the original issue's platform/architecture is

Reply:
Yes Linux and Browser are both case-sensitive so probably hitting the same issue.

Reply:
Removed both occurrences of that active issue in this file!